mappers/baseline.osw

```diff
@@ -0,0 +1,52 @@
+{
+	"seed_file": null,
+	"weather_file": null,
+	"measure_paths": [],
+	"file_paths": [],
+	"run_directory": null,
+	"steps": [
+    {
+      "measure_dir_name": "ChangeBuildingLocation",
+			"arguments": {
+        "weather_file_name": "urbanopt_scenario_tests.epw",
+        "climate_zone": "5B"
+      }	
+    },{
+      "measure_dir_name": "set_run_period",
+			"arguments": {
+        "timesteps_per_hour": 1,
+        "begin_date": "2012-06-01",
+        "end_date": "2012-06-02"
+      }	
+    },{
+			"measure_dir_name": "create_bar_from_building_type_ratios",
```

Comment:
@bgschiller this should be using some of the measures from the geojson gem, that way we end up with geometry from the geojson file

Reply:
@bgschiller create_bar_from_building_type_ratios measure should be removed (currently this would create geometry that would overwrite the geometry from urban_geometry_creation)